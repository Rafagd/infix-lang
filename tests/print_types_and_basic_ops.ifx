# A comment
void println "Hello, World!";
void println "";
void println "Testing Print Types:";
void println void;
void println null;
void println true;
void println false;
void println 1;
void println 1.1;
void println "";
void println "Testing Add:";
void println (1 + 3);
void println (15.0 + 0.5);
void println "";
void println "Testing Sub:";
void println (3 - 5);
void println (20.0 - 5.5);
void println "";
void println "Testing Mul:";
void println (2 * 6);
void println (0.5 * 5.0);
void println "";
void println "Testing Div:";
void println (10 / 5);
void println (10.0 / 5.5);
void println "";
