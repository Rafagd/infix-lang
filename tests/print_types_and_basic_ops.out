Hello, World!

Testing Print Types:
void
null
true
false
1
1.100000

Testing Add:
4
15.500000

Testing Sub:
-2
14.500000

Testing Mul:
12
2.500000

Testing Div:
2
1.818182

